web/src/stream_popover.js

```diff
@@ -71,14 +74,14 @@ export function elem_to_stream_id($elem) {
 }
 
 export function is_open() {
-    return current_stream_sidebar_elem !== undefined;
+    return Boolean(stream_popover_instance);
 }
 
 export function hide_stream_popover() {
     if (is_open()) {
-        $(current_stream_sidebar_elem).popover("destroy");
         hide_left_sidebar_menu_icon();
-        current_stream_sidebar_elem = undefined;
+        stream_popover_instance.destroy();
+        stream_popover_instance = undefined;
```

Comment:
This should be set to `null` to match the fact that it's type is `null | popover` in the declaration above.

Reply:
Fixed this before pushing back here.